src/modules/Elsa.Workflows.Core/Activities/Break.cs

```diff
@@ -24,5 +24,6 @@ protected override async ValueTask ExecuteAsync(ActivityExecutionContext context
     {
         // Send a signal to the parent scope to break out of the loop.
         await context.SendSignalAsync(new BreakSignal());
+        await context.SendSignalAsync(new EndSignal());
```

Comment:
What might be the reason for the Break activity to send an End signal?

Reply:
It is set to be used to send an End signal to Flowchart when is contained in a while.
It doesn’t look like a perfect solution indeed.